peachjam/adapters/adapters.py

```diff
@@ -290,6 +292,23 @@ def update_document(self, url):
             # the source file is the PDF version
             self.download_source_file(f"{url}.pdf", created_doc, title)
 
+        if document["taxonomy_topics"]:
+            # get topics beginning with "subject-areas"
+            topics = [
+                t for t in document["taxonomy_topics"] if t.startswith("subject-areas")
+            ]
+            if topics:
+                taxonomies = Taxonomy.objects.filter(slug__in=topics)
+                created_doc.taxonomies.filter(
+                    topic__slug__startswith="subject-areas"
+                ).delete()
```

Comment:
What about the case where an existing document previously had topics under this tree, but no longer does?

Reply:
updated :+1: